test/unit/ignoring.js

```diff
@@ -1,32 +1,90 @@
 'use strict';
 
+import helpers from '../lib/helpers';
 import skate from '../../src/skate';
 
 describe('ignoring', function () {
-  it('should ignore a flagged element', function () {
+  it('should ignore a flagged element if defined after it is inserted', function () {
     var called = 0;
+    var tag = helpers.safeTagName();
 
-    // Test attaching before.
-    document.body.innerHTML = '<div></div><div id="container-1" data-skate-ignore><div><div></div></div></div><div></div>';
-    document.getElementById('container-1').innerHTML = '<div><div></div></div>';
+    helpers.fixture(`<${tag.safe} data-skate-ignore></${tag.safe}>`);
+    skate(tag.safe, {
+      created: function () {
```

Comment:
Is it worth also defining `attached` handlers and testing that their call counts are 0, too?